quickstarts/microsoft.network/azure-dns-private-resolver/main.bicep

```diff
@@ -0,0 +1,143 @@
+@description('name of the new virtual network where DNS resolver will be created')
+param resolverVNETName string = 'dnsresolverVNET'
+
+@description('the IP address space for the resolver virtual network')
+param resolverVNETAddressSpace string = '10.7.0.0/24'
+
+@description('name of the dns private resolver')
+param dnsResolverName string = 'dnsResolver'
+
+@description('the location for resolver VNET and dns private resolver')
+param resolverLocation string = 'westcentralus' //resolver available in specific region, refer the documenation for more information https://docs.microsoft.com/en-us/azure/dns/dns-private-resolver-overview#regional-availability
```

Comment:
rename this to ```location``` - also if it's not globally available use ```@allowedValues``` to provide the list

Reply:
thanks for the review @bmoore-msft. Implemented the changes.

Reply:
@bmoore-msft , we might send default location for dnsresolver/vnet to bypass the failure for valid deployment.